### ISSUE TITLE
Minikube Status command fails with 'Error executing status template'

### ISSUE THREAD

**Issue by scrossan-crown** [Azure/vscode-kubernetes-tools#467]
When trying to run minikube status in vscode I get the following error 
```
[Running,E0209 13:07:37.450999    1868 status.go:96] Error executing status template: template: status:1:23: executing "status" at <.ClusterStatus>: can't evaluate field Clu
sterStatus in type cmd.Status
```

**Comment by itowlson** [comment]
1. What operating system are you on?
2. What version of Minikube are you running?
3. If you go to your OS command line (or the Code integrated terminal) and run `minikube status`, what do you see?

Thanks!

**Comment by brendandburns** [comment]
This looks to me like the minikube on your machine is mis-configured. I suspect that `minikube` at the CLI won't work correctly...

Did you install minikube using this extension?

Thanks

**Comment by gdamberg** [comment]
I have a similar problem. Complete error output:
`Error getting status Error: failed to get status: E0314 15:24:44.532956   17828 status.go:137] Error executing status template: template: status:1:4: executing "status" at <.MinikubeStatus>: can't evaluate field MinikubeStatus in type cmd.Status
`

Ubuntu 18.04, VS Code 1.32.1, Kubernetes extension 0.1.18

Minikube output:
`
$  minikube status
host: Running
kubelet: Running
apiserver: Running
kubectl: Correctly Configured: pointing to minikube-vm at 192.168.99.100

$  minikube version
minikube version: v0.33.1
`

**Comment by itowlson** [comment]
Looking at the code, the status command we run is:

`minikube status --format '["{{.MinikubeStatus}}","{{.ClusterStatus}}","{{.KubeconfigStatus}}"]'``

Could you try that at the CLI please?  Thanks!  @brendandburns is that format string likely still to be correct or might Minikube have made a breaking change?

**Comment by gdamberg** [comment]
`$ minikube status --format '["{{.MinikubeStatus}}","{{.ClusterStatus}}","{{.KubeconfigStatus}}"]'
["E0315 09:15:29.033089   28619 status.go:137] Error executing status template: template: status:1:4: executing "status" at <.MinikubeStatus>: can't evaluate field MinikubeStatus in type cmd.Status
`
Output of status seems to have changed at some point: https://godoc.org/k8s.io/minikube/cmd/minikube/cmd#Status

**Comment by itowlson** [comment]
Thanks @gdamberg.  @brendandburns the current Status properties are documented at https://godoc.org/k8s.io/minikube/cmd/minikube/cmd#Status - looks like they now have Host, Kubelet, ApiServer and Kubeconfig.  How do these map to what you want the status command to display?  Is there a way we can make this robust across versions?

**Comment by DerSalvador** [comment]
> Thanks @gdamberg. @brendandburns the current Status properties are documented at https://godoc.org/k8s.io/minikube/cmd/minikube/cmd#Status - looks like they now have Host, Kubelet, ApiServer and Kubeconfig. How do these map to what you want the status command to display? Is there a way we can make this robust across versions?

Yes, make the --format string configurable through ENV or config file, thanks

**Comment by itowlson** [comment]
Fixed via #575.
